index.js

```diff
@@ -131,9 +131,11 @@ class HtmlWebpackPlugin {
       ...global,
       HTML_WEBPACK_PLUGIN: true,
       require: require,
+      __non_webpack_require__: require,
       htmlWebpackPluginPublicPath: publicPath,
       URL: require('url').URL,
-      __filename: templateWithoutLoaders
+      __filename: templateWithoutLoaders,
+      __dirname: path.dirname(templateWithoutLoaders)
```

Comment:
probably we should also add a unit test for `__filename` and `__dirname` - what do you think?

Reply:
Will add a test!